scripts/get-current-branch.sh

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 # Check if short name work
-if [[ $branch == "main" || $branch =~ ^secureli[0-9][0-9][0-9]$ || $branch =~ "dependabot" ]]; then
+if [[ $branch == "main" || $branch =~ ^secureli[0-9][0-9]?[0-9]?$ || $branch =~ "dependabot" ]]; then
```

Comment:
Can we do this instead?
```suggestion
if [[ $branch == "main" || $branch =~ ^secureli[0-9]+$ || $branch =~ "dependabot" ]]; then
```

Reply:
Yes, that makes sense and is more maintainable. Thanks. Fixed!